src/bluetoothdevice.h

```diff
@@ -417,9 +452,25 @@ class bluetoothdevice : public QObject {
     void groundContactChanged(double groundContact);
     void verticalOscillationChanged(double verticalOscillation);
 
+  private:
+
+    bool virtualDeviceSetupDone = false;
```

Comment:
This replaces the "firstInit", "firstVirtualX", "firstStateChanged" flags used to determine (in most cases) if the virtual device setup has been done before.